.eslintrc.json

```diff
@@ -379,6 +379,7 @@
                 "subs": false,
                 "tab_bar": false,
                 "templates": false,
+                "theme_detection": false,
```

Comment:
This list is for JS global variables, and `theme_detection` should not need to be a global variable.